public/template/main-client.jade

```diff
@@ -22,8 +22,9 @@
           p
             a.btn.primary.large(href="/main/register") Get Started &raquo;
 
-      .span4
-        img.img-rounded(src="images/somefriends.jpg", width="340", height="235")
+      .span4.pull-center
+        if config.mainImage
+          img.img-rounded(src="#{config.mainImage}")
```

Comment:
Maybe there's nothing to be done, but AFAICT there'll be a layout reflow now when the image is loaded right? Since the `width` and `height` HTML attributes are now gone?

Reply:
Yes, the width/height in the element is a little hardcoded and force the size, the max-(width/height) fix that.

Reply:
You misunderstood me. The layout of the home page will shift when the image is loaded because the width and height of the image are now only known after the image request completes, not when the HTML loads. Unless I'm missing something?

AFAIK there isn't really anything to be done about this, but it _does_ seem unfortunate...

Reply:
No, the behavior is the same, not affect the render times, the css is loaded before the image